docs-src/go/how-to-trigger-a-schedule-in-go.md

```diff
@@ -3,6 +3,10 @@ id: how-to-trigger-a-schedule-in-go
 title: How to trigger a Schedule in Go
 sidebar_label: Trigger Schedule
 description: To trigger a Schedule in Go, use `trigger()` on `ScheduleHandle`.
+tags:
+  - schedule
+  - dacx
```

Comment:
not sure about this tag.
we don't currently use it in Assembly - 
and not sure about the external value
wdyt?